automatic/keeweb/update.ps1

```diff
@@ -1,7 +1,5 @@
 import-module au
 
-$releases = 'https://github.com/keeweb/keeweb/releases'
```

Comment:
This URL should have be used to replace the URL specified in the `VERIFICATION.txt` file instead of removing it alltogether.
Not sure why that hasn't been done, though.

Reply:
Since that URL is not used at all, there is no point in replacing the content of VERIFICATION.txt ([e.g. like in Clementine](https://github.com/chocolatey-community/chocolatey-coreteampackages/blob/e1ba7e225d190aed9bf0bce035c0b7ab64bba22c/automatic/clementine/update.ps1#L17)) each time. Any way, when that link will change we will have to change it in the nuspec file as well any way, so...